testsuite/classlibrary/TestNodeProxy_Server.sc

```diff
@@ -71,6 +71,7 @@ TestNodeProxy_Server : UnitTest {
 		// ... and also afterwards, we force an update
 		server.sendStatusMsg;
 		server.sync;
+		0.8.wait;
```

Comment:
Is this really the only way to fix this? This is a rather lengthy wait. Could this be addressed by sending some other message to the server and running another server.sync? Or maybe trying the shorter wait time? 

If you remove the exclusion from `gha_test_run_proto.json` we'll be able to see how the shorter wait time performs in the CI (which is arguably not tuned for audio/low-latency performance).

Reply:
I agree. I find it weird that this doesn't work faster, but it has to do with the time the server takes to report the deallocation of the synth def.

Reply:
I've found another solution: just fix the wrong order in the `sync` …